Code Quality task failing: unable to find pmd

I'm not sure how long this has been a problem, but I am trying to find out why UI tests are not being ran on Firebase currently (with whatever currently pass, that's another problem), but the Code Quality task is a dependency for the scheduled UI test task. 

Code Quality Failure: https://firefox-ci-tc.services.mozilla.com/tasks/OohJuDmMT-iQ9rYdBo99NA/runs/0/logs/https%3A%2F%2Ffirefox-ci-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2FOohJuDmMT-iQ9rYdBo99NA%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Flive.log

```Log
<======-------> 50% CONFIGURING [37s]> :app<=============> 100% CONFIGURING [37s]
FAILURE: Build failed with an exception.
* What went wrong:
Task 'pmd' not found in root project 'focus-android'.
```

(@pocmo ?)

--- a/tools/taskcluster/schedule-master-build.py
+++ b/tools/taskcluster/schedule-master-build.py
@@ -45,7 +45,7 @@ def generate_code_quality_task(buildTaskId):
     return taskcluster.slugId(), generate_task(
         name="(Focus for Android) Code quality",
         description="Run code quality tools on Focus/Klar for Android code base.",
-        command='echo "--" > .adjust_token && ./gradlew --no-daemon clean detektCheck ktlint lint pmd checkstyle spotbugs',
+        command='echo "--" > .adjust_token && ./gradlew --no-daemon clean detektCheck ktlint lint',
         dependencies=[buildTaskId])
 
 

